specification/index.bs

```diff
@@ -190,7 +199,70 @@ Issue(62): Specify localization handling.
 
 ### Content scripts
 
-#### Isolated worlds
+<dfn>Content scripts</dfn> represent a set of JS and CSS files that should be injected into pages loaded by the user agent.
+
+#### Key `matches`
+
+A [=list=] of [=match patterns=] that are used to decide where the content script runs. This key is required.
+
+#### Key `exclude_matches`
+
+A [=list=] of [=match patterns=] that should be used to exclude URLs from where the content script runs.
+
+#### Key `js`
+
+A [=list=] of file paths that should be injected as scripts.
+
+#### Key `css`
+
+A [=list=] of file paths that should be injected as stylesheets.
+
+#### Key `all_frames`
+
+If `all_frames` is true, the content script must be injected into subframes. Defaults to false.
+
+#### Key `match_about_blank`
+
+If this is `true`, the content script will also be injected into an additional user agent specified set of pages used to represent empty frames. This will only happen if the content script matches the page that embedded the frame. Defaults to `false`.
+
+#### Key `match_origin_as_fallback`
+
+Used to match frames with an opaque or otherwise missing origin. The origin to match against is determined in the following order of priority:
+
+1. If the frame has an [=opaque origin=], such as with a [=blob URLs=], use the non-opaque origin.
+1. If available, use the origin of the parent frame.
+1. Otherwise, no origin is found and this frame can never be matched.
+
+#### Key `run_at`
+
+Specifies when the content script should be injected. Valid values are `document_start`, `document_end` and `document_idle`.
+
+#### Key `include_globs`
+
+A list of [=globs=] that a page should match in addition to matches.
```

Comment:
```suggestion
A list of [=globs=] that a page should match. A page matches if the URL matches both the [=matches=] field and the [=globs=] field.
```

(Feel free to tweak; as written, it's ambiguous if this is an additional set to inject into or an additional requirement)

Reply:
That's fair, updated :)